### DISCUSSION TITLE
Eddiehub on social media ?

### DISCUSSION THREAD

**Discussion by imvisp** [EddieHubCommunity/support#1202] [Ideas] [answered]
can we create a page on social media for eddiehub ?
like on
- instagram
- twitter
- linkedin

**Comment by Vyvy-vi** [comment, marked as answer]
Idk if we should.
- Thoughts on Twitter page:
That would create multiple sources of communication(Eddie's accounts + EddieHub's pages), which would be sorta confusing and hard to manage.
I'm not sure if Eddie would be able to manage them... Moreover, imho the community members(and the large amount of activity on twitter with the hashtag- #EddieHub) contextualise lot of the conversation.

- Thoughts on Instagram:
Maybe we can have an Instagram page, but what would that be for?

- Thoughts on Linkedin
EddieHub is already an org on LinkedIn(thus I guess we can post articles under the org)

**Reply by imvisp** [reply]
Yes Instagram and linkedin are best 👍

Yes we should write post or blog on LinkedIn

**Reply by NIKU-SINGH** [reply]
--I have a suggestion for Instagram.
--A page where all of our community members can post about other creative stuff they do, surely there are a lot of creative folks out here moreover it can be a great place to know each other too apart from the technical stuff.

**Reply by Vyvy-vi** [reply]
We do have github repos for sharing creative stuff 
https://github.com/EddieHubCommunity/Hackathons
https://github.com/EddieHubCommunity/Branding

**Reply by DevCassie** [reply]
I'm more thinking about how it would be arranged if the community members have access to the accounts. Not that I have anything against it. But it's a risk, to be honest.

**Reply by eddiejaoude** [reply]
Lots of great ideas 👍 . Sorry I am late to the party 🤓 

We now have a LinkedIn, I don't think I can manage more social at the moment. 

We could have a community blog posts on linkedin or another platform. Would people write for our community blog?

**Reply by schmelto** [reply]
> We could have a community blog posts on linkedin or another platform. Would people write for our community blog?

I think yes maybe we can make a create-content discord chat in which we discuss blog topics and then if someone wants to wrote a blob post about a several topic he can post a discussion here so everyone can review and discuss the post before we post it on LinkedIn or somewhere else

**Reply by WebShivam** [reply]
I agree with you. I think doing so will encourage more people to start writing articles and reviewing before posting it anywhere will help them to overcome the fear of writing quality (specially if they're beginners)
